IDE/Renesas/e2studio/RX65N/GR-ROSE/test/src/key_data.c

```diff
@@ -90,56 +90,56 @@ const uint32_t              encrypted_user_key_type =
 
 const unsigned char ca_ecc_cert_der_sig[] =
 {
-    0x81, 0x87, 0x55, 0x02, 0x9c, 0xf8, 0x0b, 0x5c, 0x69, 0xba, 0x4b, 0xe9, 
-    0x51, 0x19, 0x7b, 0x3c,	0x22, 0xaf, 0x6b, 0xd5, 0x23, 0x80, 0x64, 0xb8,
-    0x1e, 0xe7, 0x99, 0xab, 0x89, 0x6d, 0xff, 0x93,	0xe1, 0xb5, 0xd8, 0x8d,
-    0x7f, 0x53, 0xe7, 0x74, 0x8a, 0xd6, 0x06, 0x6a, 0x61, 0x2a, 0xab, 0x4a,
-	0x65, 0x0d, 0x28, 0xbd, 0xc7, 0x92, 0x70, 0xd8, 0x79, 0xdc, 0xf7, 0xd2,
-    0xf7, 0xf9, 0xa9, 0x13,	0x16, 0x5f, 0x4d, 0x4b, 0xae, 0x12, 0x5b, 0xfe,
-    0x08, 0x09, 0x26, 0x1f, 0xae, 0x15, 0xd9, 0x16,	0x51, 0x4a, 0x92, 0xfe, 
-    0x61, 0xeb, 0x6a, 0x82, 0x8c, 0xf8, 0xa8, 0x94, 0x26, 0x81, 0x9b, 0xb8,
-	0x74, 0xd6, 0x32, 0x2b, 0x6f, 0xa6, 0xad, 0x9a, 0x91, 0x63, 0xe6, 0x03,
-    0x5f, 0x25, 0xee, 0xc2,	0xfd, 0x54, 0x58, 0x13, 0xee, 0x53, 0xae, 0xe3, 
-    0xda, 0x6b, 0xb5, 0x08, 0x45, 0xbe, 0xf2, 0xa8,	0x22, 0x92, 0x53, 0x8c,
-    0xa4, 0xd9, 0xb6, 0x4e, 0xb9, 0xbf, 0x76, 0xb0, 0x96, 0x9f, 0xd7, 0xf6,
-	0x53, 0xf3, 0x65, 0xf0, 0xe7, 0xff, 0xe8, 0x93, 0x2d, 0xc2, 0x08, 0x91,
-    0xf8, 0x27, 0x33, 0x92,	0x95, 0x23, 0x1e, 0x6e, 0xd1, 0x91, 0x50, 0xc6,
-    0x06, 0xb1, 0x16, 0x69, 0xee, 0xcf, 0xad, 0xc9,	0x4b, 0xf2, 0x38, 0x65,
-    0x1d, 0xa9, 0xff, 0xd3, 0xf4, 0xd5, 0x84, 0xa9, 0xdd, 0xa7, 0x04, 0x77,
-	0x64, 0xb5, 0xed, 0xe8, 0x0c, 0x15, 0x77, 0x7e, 0xfc, 0x02, 0xdd, 0x4e,
-    0xf1, 0xd1, 0x36, 0xfe,	0xac, 0x3f, 0x5c, 0xd3, 0xdf, 0x51, 0xa3, 0x5c,
-    0x8d, 0xed, 0x9f, 0x8d, 0xc8, 0xa7, 0xff, 0x46,	0xbc, 0x1d, 0x83, 0x12,
-    0x4f, 0x49, 0xaa, 0xf5, 0xd1, 0x04, 0xc8, 0xee, 0x95, 0xda, 0xa0, 0x61,
-	0x28, 0xff, 0x25, 0x8c, 0xfc, 0x35, 0x8f, 0xdc, 0x00, 0x47, 0x01, 0xfd,
-    0x7c, 0xed, 0x05, 0xff
+	0x29, 0xaf, 0x9a, 0x3a, 0xc3, 0xa4, 0x21, 0x87, 0x59, 0xc7, 0x67, 0x85, 

```

Comment:
Please remove hard tabs in ca_ecc_cert_der_sig[].

Reply:
Replaced tabs with spaces.